### PR TITLE
Update dumpling `--row` description

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]
LGTM. Great job!!

File: dumpling-overview.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 默认情况下，导出的文件会存储到 `./export-<current local time>` 目录下。常用选项如下：
 
 - `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度提高导出速度，但也会加大数据库内存消耗，因此不宜设置过大。
-- `-r` 选项用于指定单个文件的最大记录数，或者说，数据库中的行数。开启后 Dumpling 会开启表内并发，提高导出大表的速度。
+- `-r` 选项用于指定单个文件的最大记录数，或者说，数据库中的行数。开启后 Dumpling 会开启表内并发，提高导出大表的速度。当上游为 TiDB 且版本为 v3.0 或更新时，该参数大于 0 表示使用 TiDB region 信息进行划分，具体取值将不再生效。
```
Comment (hfxsd) [inline dumpling-overview.md:262]:
```suggestion
- `-r` 选项用于指定单个文件的最大记录数，或者说，数据库中的行数。开启后 Dumpling 会开启表内并发，提高导出大表的速度。当上游为 TiDB 且版本为 v3.0 或更新版本时，该参数大于 0 表示使用 TiDB region 信息进行划分，具体取值将不再生效。
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -346,7 +346,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --case-sensitive | table-filter 是否大小写敏感 | false，大小写不敏感 |
 | -h 或 --host| 连接的数据库主机的地址 | "127.0.0.1" |
 | -t 或 --threads | 备份并发线程数| 4 |
-| -r 或 --rows | 将 table 划分成 row 行数据，一般针对大表操作并发生成多个文件。|
+| -r 或 --rows | 将 table 划分成 row 行数据，一般针对大表操作并发生成多个文件。当上游为 TiDB 且版本为 v3.0 或更新时，该参数大于 0 表示使用 TiDB region 信息进行划分，具体取值将不再生效 |
```
Comment (hfxsd) [inline dumpling-overview.md:349]:
```suggestion
| -r 或 --rows | 将 table 划分成 row 行数据，一般针对大表操作并发生成多个文件。当上游为 TiDB 且版本为 v3.0 或更新版本时，该参数大于 0 表示使用 TiDB region 信息进行划分，具体取值将不再生效。 |
```

File: dumpling-overview.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ dumpling \
 - `-h`、`-P`、`-u` 分别代表地址、端口、用户。如果需要密码验证，可以使用 `-p $YOUR_SECRET_PASSWORD` 将密码传给 Dumpling。
 - `-o` 用于选择存储导出文件的目录，支持本地文件路径或[外部存储 URL](/br/backup-and-restore-storages.md) 格式。
 - `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度提高导出速度，但也会加大数据库内存消耗，因此不宜设置过大。一般不超过 64。
-- `-r` 用于指定单个文件的最大行数，指定该参数后 Dumpling 会开启表内并发加速导出，同时减少内存使用。
+- `-r` 用于指定单个文件的最大行数，指定该参数后 Dumpling 会开启表内并发加速导出，同时减少内存使用。当上游为 TiDB 且版本为 v3.0 或更新版本时，该参数大于 0 表示使用 TiDB region 信息进行划分，具体取值将不再生效。
```
Comment (shichun-0415) [inline dumpling-overview.md:84]: 使用 TiDB region 信息进行划分------划分什么呢？

Comment (lance6716) [reply dumpling-overview.md:84]: 划分表内并发

Comment (shichun-0415) [reply dumpling-overview.md:84]: @lance6716 @hfxsd 可以改为“该参数大于 0 表示根据 TiDB region 信息分配表内并发”吗？划分应该是分配的意思吧？

Comment (hfxsd) [reply dumpling-overview.md]: 有更新了一点儿。

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**